### PR TITLE
Add view chorals page

### PR DESCRIPTION
**Pull request by cpjk**
Also:

- Add view helpers module for collecting view helper functions. Pass to views for access to the functions.

chorals2

### CODE REVIEW

**Comment by andreirtaylor** [comment]
Wow this looks really good.

**Comment by andreirtaylor** [comment]
I think we should stop making a new file for every route. Maybe put this into routes/index.js instead.

File: views/chorals.pug [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          td= viewHelpers.truncateWithDots(choral.choralId, 30)
+          td= choral.sampleRate
+          td= choral.func
+  // Footer
+  footer.w3-container.w3-padding-16.w3-light-grey
+    h4 Powered by Choral Cloud
+    p
+      a.w3-btn.w3-black(href='https://github.com/ChoralCloud', target='_blank')
+        i.fa.fa-code-fork.w3-large
+        |  Check out the code
```
Comment (andreirtaylor) [inline views/chorals.pug:33]: What is this for?

**Comment by cpjk** [comment]
> I think we should stop making a new file for every route. Maybe put this into routes/index.js instead.

I think it would be cleaner if we put related routes in a single file.

the newChoral route code will go in here as well.

**Comment by andreirtaylor** [comment]
So new chorals will be under "chorals/newchorals"

**Comment by andreirtaylor** [comment]
Should we add a entry to the sidebar for this page?

**Comment by cpjk** [comment]
> So new chorals will be under "chorals/newchorals"

No, I just put all the chorals/* routes in `chorals.js`

e.g. `GET /chorals`, `GET /chorals/new`, `POST /chorals/`

**Comment by cpjk** [comment]
> Should we add a entry to the sidebar for this page?

It's already there. You added it IIRC. -> "chorals"

**Comment by andreirtaylor** [comment]
This looks good to me if @jboretsky or @ecuellog are good with it then so am I.

File: views/chorals.pug (hunk 1 above)
Comment (cpjk) [reply views/chorals.pug:33]: Sry didn't see this. This is just boilerplate. It's the page footer. Should move this to the layout actually.

File: views/chorals.pug [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        td Function
+      each choral in chorals
+        tr
+          td= choral.name
+          td= choral.choralType
+          td= viewHelpers.truncateWithDots(choral.choralId, 30)
+          td= choral.sampleRate
+          td= choral.func
+  // Footer
+  footer.w3-container.w3-padding-16.w3-light-grey
```
Comment (andreirtaylor) [inline views/chorals.pug:28]: I think you need to remove this from this file. There are two powered by choral sections on the page.

**Pull request merged**